docs/user/reference/policies.rst

```diff
@@ -1,16 +1,20 @@
-Legal Documents and Policies
+============================
+Policies and legal documents
 ============================
 
 Here is some of the fine print used by |org_brand| and |com_brand|:
 
 .. toctree::
-    :glob:
-    :hidden:
```

Comment:
Seems we're repeating the TOC again here by removing hidden? 

https://docs--9916.org.readthedocs.build/en/9916/reference/policies.html
![Screenshot 2023-01-18 at 4 26 08 PM](https://user-images.githubusercontent.com/25510/213327050-fb0b962f-93f7-404a-8a11-cfd1f97efabb.png)

Reply:
You're right, the old version looked sloppy and putting efforts into a proper little "definition list" makes the page much more pleasant to look at.